Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,49 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "without-test", "Skip running build-time tests (not recommended)"
+  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+
+  depends_on "robodoc" => [:recommended, :build]
+  depends_on "cmake" => :build
+  depends_on :fortran
+
+  def install
+    mktemp do
+      args = ["-DUSE_GNU_INSTALL_CONVENTION:BOOL=TRUE"]
+      args << "-DENABLE_UNICODE:BOOL=TRUE" if build.with? "unicode-support"
+      args << "-DSKIP_DOC_GEN:BOOL=TRUE" if build.without? "robodoc"
+      system "cmake", buildpath, *(args + std_cmake_args)
+      if build.with? "test"
+        system "make"
+        system "make", "test"
+      end
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"json_test.f90").write <<-EOS.undent
+      program json_test
+        use json_module
+        use ,intrinsic :: iso_fortran_env ,only: error_unit
+        implicit none
+        call json_initialize()
+        if ( json_failed() ) then
+          call json_print_error_message(error_unit)
+          stop 2
+        endif
+      end program
+    EOS
+
+    fc = ENV["FC"] || "#{Formula["gcc"].opt_bin}/gfortran"
+    system fc, "-ljsonfortran", "-I#{HOMEBREW_PREFIX}/include", "#{testpath}/json_test.f90"
```

Comment:
This is basically what `ENV.fortran` does already, so you should be able to just use
```
ENV.fortran
system ENV.fc, "-ljsonfortran", "-I#{include}", "json_test.f90"